src/routes/components/components.json

```
@@ -2981,5 +2981,210 @@
 		"category": "Data Visualisation",
 		"tags": ["components and libraries", "images"],
 		"stars": 0
+	},
+	{
+		"title": "svelte-transition",
+		"url": "https://captaincodeman.github.io/svelte-headlessui/transition/",
```

Since a few minutes, `repository` should be used instead of `url`.
`url` can still be used, but it's more for a homepage or a documentation rather than a source repository.

_(This comment is applicable for all components in this PR)_
```suggestion
		"repository": "https://captaincodeman.github.io/svelte-headlessui/transition/",
```

---

I review this PR too fast!

But still, some parts are still valid.

You can (and should) use `repository` to link to the source code repository.
(The `repository` field is used to update the number of stars display on the website)